saleor/order/utils.py

```diff
@@ -113,27 +113,25 @@ def add_variant_to_order(
         order, variant, total_quantity) if add_to_existing else total_quantity
     price = variant.get_price_per_item(discounts)
     while quantity_left > 0:
```

Comment:
This `while` loop was useful when we had multiple stocks that we had to iterate over to check quantity in each of them. Do we still need it? Maybe this function could be simplified a bit.